src/app/demo/typescript/typescript.component.ts

```diff
@@ -10,6 +10,36 @@ import {ng2tsConfig} from '../../../../ng2ts/ng2ts';
 })
 
 export class TypescriptComponent {
+
+  introJsBefore() {
+    let squiggly;
+
+    squiggly = document.querySelector('.highlighted-code');
```

Comment:
I don't know why but it wasn't working before in one line; maybe it will work now. 😂😭